fern/fern.config.json

```diff
@@ -1,4 +1,4 @@
 {
   "organization": "octoai",
-  "version": "0.19.9"
+  "version": "0.19.19"
```

Comment:
What's this for?

Reply:
My attempt at updating the generator and also making the fern cli client stop telling me it can be updated. I don't think it is necessary.